.github/workflows/build.yml

```diff
@@ -320,11 +320,10 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
         with:
           components: clippy
-      - run: cargo clippy --locked --package rustls --all-features --all-targets -- --deny warnings
+        # because examples enable rustls' features, `--workspace --no-default-features` is not
+        # the same as `--package rustls --no-default-features` so run it separately
       - run: cargo clippy --locked --package rustls --no-default-features --all-targets -- --deny warnings
-      - run: cargo clippy --locked --manifest-path=connect-tests/Cargo.toml --all-features --all-targets -- --deny warnings
-      - run: cargo clippy --locked --manifest-path=fuzz/Cargo.toml --all-features --all-targets -- --deny warnings
```

Comment:
I think this invocation for `--manifest-path=fuzz/Cargo.toml` needs to stay. That crate is a little funky in that it's not included in the top-level `Cargo.toml` workspace members.

If i introduce an issue into one of the fuzzers running `cargo clippy --locked --workspace --all-features --all-targets -- --deny warnings` doesn't flag it, but running `cargo clippy --locked --manifest-path=fuzz/Cargo.toml --all-features --all-targets -- --deny warnings` does.

Reply:
thanks for catching that. fixed